middlewares/route-validator.js

```diff
@@ -54,4 +65,38 @@ module.exports = {
       check('token').not().isEmpty(),
     ],
   },
+  admin: {
+    createModerator: [
+      check('nickname')
+        .not().isEmpty().withMessage('nickname is required')
+        .isString()
+        .custom(isUnique('nickname'))
+        .withMessage('nickname must be unique'),
+      check('name')
+        .not().isEmpty()
+        .isString(),
+      check('surname')
+        .not().isEmpty()
+        .isString(),
+      check('birth_date')
+        .not().isEmpty().withMessage('Birth date is required')
+        .matches(datePattern)
+        .withMessage('Date must be yyyy-mm-dd format'),
+      check('email')
+        .not().isEmpty().withMessage('E-mail is required')
+        .isEmail()
+        .withMessage('email bad is bad formatted')
+        .custom(isUnique('email'))
+        .withMessage('E-mail must be unique'),
+      check('password', empty)
+        .not().isEmpty()
+        .isString(),
+    ],
+    checkUniqueField: [
```

Comment:
Better to write functions for such operations .
For example
function isEmpty(filed){ return check(field, empty).not().isEmpty(); } checUniqueFiled: [ isEmpty('filed'), isEmpty('value')]
1 - easier supporting
3 - DRY.